### ISSUE TITLE
java.lang.IllegalArgumentException: Unknown pattern character 'Y'

### ISSUE THREAD

**Issue by kateile** [ParkSangGwon/TedImagePicker#6]
Whenever Picker is opened and I try to scroll down I get this exception. I am using it in Fragment
as 
```
    private fun openGallery() {
        TedImagePicker.with(requireContext()).start(object : OnMultiSelectedListener {
            override fun onSelected(uriList: List<Uri>) {
                images.clear()
                images.addAll(uriList)
                thumbnailsAdapter.notifyDataSetChanged()
            }
        })
    }
```
Exception:
```
E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.myAmazingApp, PID: 30823
    java.lang.IllegalArgumentException: Unknown pattern character 'Y'
        at java.text.SimpleDateFormat.validatePatternCharacter(SimpleDateFormat.java:314)
        at java.text.SimpleDateFormat.validatePattern(SimpleDateFormat.java:296)
        at java.text.SimpleDateFormat.<init>(SimpleDateFormat.java:356)
        at gun0912.tedimagepicker.TedImagePickerActivity$setupMediaRecyclerView$$inlined$run$lambda$1.onScrolled(TedImagePickerActivity.kt:210)
        at androidx.recyclerview.widget.RecyclerView.dispatchOnScrolled(RecyclerView.java:5077)
        at androidx.recyclerview.widget.RecyclerView.scrollByInternal(RecyclerView.java:1975)
        at androidx.recyclerview.widget.RecyclerView.onTouchEvent(RecyclerView.java:3297)
        at android.view.View.dispatchTouchEvent(View.java:8149)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2416)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2140)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2422)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2155)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2422)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2155)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2422)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2155)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2422)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2155)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2422)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2155)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2422)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2155)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2422)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2155)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2422)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2155)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2422)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2155)
        at android.view.ViewGroup.dispatchTransformedTouchEvent(ViewGroup.java:2422)
        at android.view.ViewGroup.dispatchTouchEvent(ViewGroup.java:2155)
        at com.android.internal.policy.impl.PhoneWindow$DecorView.superDispatchTouchEvent(PhoneWindow.java:2295)
        at com.android.internal.policy.impl.PhoneWindow.superDispatchTouchEvent(PhoneWindow.java:1622)
        at android.app.Activity.dispatchTouchEvent(Activity.java:2565)
        at androidx.appcompat.view.WindowCallbackWrapper.dispatchTouchEvent(WindowCallbackWrapper.java:69)
        at androidx.appcompat.view.WindowCallbackWrapper.dispatchTouchEvent(WindowCallbackWrapper.java:69)
        at com.android.internal.policy.impl.PhoneWindow$DecorView.dispatchTouchEvent(PhoneWindow.java:2243)
        at android.view.View.dispatchPointerEvent(View.java:8357)
        at android.view.ViewRootImpl$ViewPostImeInputStage.processPointerEvent(ViewRootImpl.java:4778)
        at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:4644)
        at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4202)
        at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4256)
        at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4225)
        at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:4336)
        at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4233)
        at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:4393)
        at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4202)
        at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4256)
        at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4225)
        at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4233)
        at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4202)
    	at android.view.ViewRootImpl.delive
```

**Comment by DennisGuo** [comment]
I have the same issue with Android 5.1.1 ,API 22

**Comment by rajendrachauhan** [comment]
I have the same issue with Android 6.0.1 ,API 23.
IllegalArgumentException: Unknown pattern character 'Y'
        at java.text.SimpleDateFormat.validatePatternCharacter(SimpleDateFormat.java:323)
        at java.text.SimpleDateFormat.validatePattern(SimpleDateFormat.java:305)
        at java.text.SimpleDateFormat.<init>(SimpleDateFormat.java:365)
        at gun0912.tedimagepicker.TedImagePickerActivity$setupMediaRecyclerView$$inlined$run$lambda$1.onScrolled(TedImagePickerActivity.kt:210)
Whenever scroll down to get this issue.
Please check this issue and update lib.
and lib is very easy and usefull.

**Comment by lmy4080** [comment]
You can fix this issue by adding this function

 TedImagePicker.with(this)
                    .scrollIndicatorDateFormat("yyyy.MM")

**Comment by ParkSangGwon** [comment]
Thanks all guys!
This issue fixed at 1.0.5
